### ISSUE TITLE
Apps mute and volume features work very unexpectedly with WASAPI disabled

### ISSUE THREAD

**Issue by XLTechie** [nvaccess/nvda#16409]
### Steps to reproduce:

1. IN NVDA Advanced settings, set WASAPI to disabled.
2. Apply, save config, and restart NVDA.
3. Start some sound playing. I used Firefox, playing a video.
4. Attempt to mute other apps (`NVDA+Alt+Del`).
5. Attempt to change volume of other apps (`NVDA+Alt+pgDn`).
6. Continue lowering volume of other apps to 20 or so.
7. In NVDA Advanced settings, enable WASAPI.
8. Save config and restart NVDA.

#### At step 4, NVDA says:

>Sound split cannot be used. Please enable WASAPI in the Advanced category in NVDA Settings to use it.
>Applications muted

First,. I was not trying to use Sound Split. Why is it being mentioned?
Second, Applications have not been muted, so the message is false.

#### At step 5, NVDA says:

>Sound split cannot be used. Please enable WASAPI in the Advanced category in NVDA Settings to use it.
>Applications volume 95

First, I was not using Sound Split, again.
Second, the application volume was not changed, so the message is false.

#### At step 8:

Upon NVDA restart, the volume of the other application--which was previously not changed--suddenly drops precipitously to match the volume that was set while WASAPI was disabled.

### Expected behavior:

At step 4, NVDA should speak only something like:

>Application mute is only supported when WASAPI is enabled in NVDA advanced settings.

At step 5, NVDA should speak only something like:

>Application volume changes are only supported when WASAPI is enabled in NVDA advanced settings.

At step 8, the volume should not change on NVDA startup, because step 4 should not have changed any volume in configuration.

#### NVDA installed/portable/running from source:

Portable.

#### NVDA version:
alpha-31622,11a2b4f1

#### Windows version:
11 HOME 23H3

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your computer?
N/A

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Of course.

**Comment by XLTechie** [comment]
cc @mltony
Also @CyrilleB79 as a possibly interested party.

**Comment by Adriani90** [comment]
As far as I know these features are designed to work well only with WASAPI, and the setting to disable WASAPI might be completely removed from the GUI. What is the reason you are not using WASAPI?
Cc: @jcsteh

**Comment by XLTechie** [comment]
@Adriani90 I'm pretty sure that doesn't matter.
As long as WASAPI CAN be disabled, other features should work correctly in that scenario.

I don't think users should have to justify using a core NVDA feature that is not even scheduled for removal.

That said, I was not using WASAPI in that particular portable alpha, because I was previously testing an add-on that doesn't work with WASAPI.

**Comment by jcsteh** [comment]
> As long as WASAPI CAN be disabled, other features should work correctly in that scenario.

That might be true if disabling WASAPI were not an advanced setting, but on the contrary, it is an advanced setting. The advanced settings section has a very clear disclaimer about the potential hazards of changing these settings: "I understand that changing these settings may cause NVDA to function incorrectly."

> I don't think users should have to justify using a core NVDA feature that is not even scheduled for removal.

On the contrary, they should - and they accept the associated risk - because it's an advanced setting.

All of that said, it looks like this is being addressed in #16404 anyway.

**Comment by seanbudd** [comment]
Our plan is to revert this feature
